package.json

```diff
@@ -61,6 +61,7 @@
     "@digitalservice4germany/style-dictionary": "~2.0.0",
     "@eslint/compat": "~1.2.2",
     "@eslint/js": "~9.14.0",
+    "@iconify-json/ic": "^1.2.1",
```

Comment:
Question: Can we not get this icon from the other icon packs?

Reply:
I checked, and they do have similar icons. However, this one had the best weight, so I decided to pull in just one more 😅